### ISSUE TITLE
Remapping 'd' and 'c' makes leap don't work.

### ISSUE THREAD

**Issue by phanen** [ggandor/leap-spooky.nvim#16]
Thanks for your awesome work.

I often use black hole register, so I remap:
```lua
vim.keymap.set('n', 'd', '"_d');
vim.keymap.set('n', 'c', '"_c');
```
But it seems not work with this plugin.

**Comment by ggandor** [comment]
What do you mean "not work"? What happens (or doesn't happen)? I just remapped `d` to `"_d` and don't see any problem, spooky actions work fine.

**Comment by phanen** [comment]
Sorry, I mixed things up, spooky will work well if I set a longer `timeoutlen`.
As for keymap, It is basically a [problem of whick-key](https://github.com/folke/which-key.nvim/issues/271):
- if `vim.keymap.set('n', 'd', '"_d')`, type `d` will not trigger popup...

I learn that spooky works by mapping keys like `rr` `ir` to textobjects. It's wonderful and minimal, but we still need fast enough typing. A long `timeoutlen` (or a `which-key` without the problem of popup) can be a workaround now.
But It would be better if there's a indefinite pending between `r` and `r`, or `i` and `r`. (like issue #15)

So I'll close this issue.
